yt_dlp/extractor/la7.py

```diff
@@ -74,9 +79,7 @@ def _real_extract(self, url):
         formats.extend(m3u8_formats)
 
         for q in filter(None, video_path.split(',')):
-            http_f = self._generate_mp4_url(q, m3u8_formats)
-            if http_f:
-                formats.append(http_f)
+            formats.append(self._generate_mp4_url(q, m3u8_formats))
 
         self._sort_formats(formats)
```

Comment:
```suggestion
```
This is deprecated

Reply:
but then the formats are presented to the user as they are added to the formats list. is that ok?

sorted:
```
[info] Available formats for il-gol-di-lindsey-thomas-fiorentina-vs-milan-serie-a-calcio-femminile-26-11-2022-461736:
ID            EXT RESOLUTION FPS |  FILESIZE   TBR PROTO | VCODEC        VBR ACODEC     ABR ASR MORE INFO
--------------------------------------------------------------------------------------------------------------------
dash-f1-a1-x3 m4a audio only     |             48k dash  | audio only        mp4a.40.2  48k 44k DASH audio, m4a_dash
dash-f2-a1-x3 m4a audio only     |             95k dash  | audio only        mp4a.40.2  95k 48k DASH audio, m4a_dash
dash-f1-v1-x3 mp4 640x360     25 |            617k dash  | avc1.42c01e  617k video only         DASH video, mp4_dash
hls-664       mp4 640x360     25 |            665k m3u8  | avc1.42c01e  665k mp4a.40.2   0k
https-664     mp4 640x360     25 | ~ 2.27MiB  665k https | avc1.42c01e  665k mp4a.40.2   0k
dash-f2-v1-x3 mp4 1280x720    25 |           3522k dash  | avc1.4d4028 3522k video only         DASH video, mp4_dash
hls-3617      mp4 1280x720    25 |           3617k m3u8  | avc1.4d4028 3617k mp4a.40.2   0k
https-3617    mp4 1280x720    25 | ~12.25MiB 3617k https | avc1.4d4028 3617k mp4a.40.2   0k
```
not sorted:
```
[info] Available formats for il-gol-di-lindsey-thomas-fiorentina-vs-milan-serie-a-calcio-femminile-26-11-2022-461736:
ID            EXT RESOLUTION FPS |  FILESIZE   TBR PROTO | VCODEC      ACODEC     ASR MORE INFO
----------------------------------------------------------------------------------------------------------
dash-f1-v1-x3 mp4 640x360     25 |            617k dash  | avc1.42c01e video only     DASH video, mp4_dash
dash-f2-v1-x3 mp4 1280x720    25 |           3522k dash  | avc1.4d4028 video only     DASH video, mp4_dash
dash-f1-a1-x3 m4a audio only     |             48k dash  | audio only  mp4a.40.2  44k DASH audio, m4a_dash
dash-f2-a1-x3 m4a audio only     |             95k dash  | audio only  mp4a.40.2  48k DASH audio, m4a_dash
hls-664       mp4 640x360     25 |            665k m3u8  | avc1.42c01e mp4a.40.2
hls-3617      mp4 1280x720    25 |           3617k m3u8  | avc1.4d4028 mp4a.40.2
https-664     mp4 640x360     25 | ~ 2.27MiB  665k https | avc1.42c01e mp4a.40.2
https-3617    mp4 1280x720    25 | ~12.25MiB 3617k https | avc1.4d4028 mp4a.40.2
```

Reply:
Format sorting is now done in core code. Merge with master to see

Reply:
ok, done.